src/inmanta/plugins.py

```diff
@@ -191,41 +191,41 @@ def clear(cls, inmanta_module: Optional[str] = None) -> None:
             cls.__functions = {}
 
 
-class Never(inmanta_type.Type):
+class Null(inmanta_type.Type):
```

Comment:
I would also implement the `type_string`  method, as this is used to format errors

Reply:
Done